joomla/joomla-cms
# [5.0] Remove "uninstallRepeatableFieldsPlugin" method from script.php

Pull Request for Issue # .

### Summary of Changes

This pull request (PR) removes the method "uninstallRepeatableFieldsPlugin" and calls to it from file "administrator/components/com_admin/script.php".

The method once had been added with PR #28319 for updating from 3.10 to 4.x and later been extended by diverse PR's for migration of repeatable fields to subforms.

The PR is one in a sequence of PR's for removing obsolete actions from script.php which are not relevant anymore because updates to 5.x will have to be done from a 4.4 version, like it is now with 4.x from 3.10.

### Testing Instructions

Code review: Make sure that the removed method is not called anywhere else and that it does something which is definitely not needed when updating from 4.4 to 5.

Optional real test: Make sure that updating the latest 4.4-dev nightly build to the update package created by drone works when using the upload & update method.

### Actual result BEFORE applying this Pull Request

Obsolete code for converting repeatable fields to subforms and removing the repeatable fields plugin when updating from 3.10 to 4.x is present in script.php.

### Expected result AFTER applying this Pull Request

Obsolete code for converting repeatable fields to subforms and removing the repeatable fields plugin when updating from 3.10 to 4.x is not present anymore in script.php.

Updating from a 4.4-dev nightly build works.

### Link to documentations
Please select:
